core/src/main/scala/com/nvidia/spark/rapids/tool/qualification/QualificationMain.scala

```diff
@@ -60,6 +60,7 @@ object QualificationMain extends Logging {
     val reportSqlLevel = appArgs.perSql.getOrElse(false)
     val platform = appArgs.platform.getOrElse("onprem")
     val mlOpsEnabled = appArgs.mlFunctions.getOrElse(false)
+    val ignoreTransitions = appArgs.penalizeTransitions.getOrElse(true)
```

Comment:
can we change the name here to penalizeTransitions... because I would expect that to be true but ignoreTransitions to be false.. ie they are the inverse